test/unit/src/audio/FftUnit.cpp

```diff
@@ -3,7 +3,7 @@
 // FIXME: OOURA roundtrip FFT seems to be broken on windows for sizeFft = 4 (https://github.com/cinder/Cinder/issues/1263)
 #if ! defined( CINDER_MSW )
 
-#include "catch.hpp"
+#include "../catch.hpp"

```

Comment:
If you want to move catch.hpp somewhere more sensible, I'm fine with that.  Where do you want it?

Or are suggesting adding a -I .. when building the subdirs?  That'd be kinda ugly.